src/main/java/org/folio/circulation/rules/Drools.java

```diff
@@ -69,11 +71,11 @@ private KieSession createSession(MultiMap params, Location location) {
    * @param location - location with institution, library and campus
    * @return the name of the loan policy
    */
-  public String loanPolicy(MultiMap params, Location location) {
+  public Pair<String, Integer> loanPolicy(MultiMap params, Location location) {
```

Comment:
I think this should return something more expressive than a pair of string and integer (similarly for the other methods)